kDriveCore/AudioPlayer/SingleTrackPlayer.swift

```diff
@@ -87,6 +90,33 @@ public final class SingleTrackPlayer {
         reset()
     }
 
+    private func extractTrackMetadata(from asset: AVAsset) -> TrackMetadata {
```

Comment:
This can be costly. Use async to do it in the backgorund

Reply:
Yup, that fixed in the second PR